taskcluster_util/taskcluster_download.py

```diff
@@ -92,6 +92,7 @@ def _configure_login(self):
             logging_config = LOGGING_POLICY['no_logs']
         else:
             logging_config = LOGGING_POLICY['default']
+            logging.getLogger('requests').setLevel(logging.WARNING)
```

Comment:
For removing log `INFO: Starting new HTTPS connection (1): index.taskcluster.net` from requests